AirCasting/SessionsSynchronization/Controller/Utils/SessionsSynchronizationStoreDataConverter.swift

```diff
@@ -0,0 +1,63 @@
+// Created by Lunar on 16/06/2021.
+//
+
+import Foundation
+
+/// Provides conversion functions between various sessions sync data structures
+struct SynchronizationDataConterter {
+    @available(*, unavailable, message: "This struct is not meant to be instantiated. It only provides static functions")
+    private init() { }
```

Comment:
caseless enum ?

Reply:
But imo this should be a standard class with regular methods. This is logic is not a universal defined, never changing logic and I see lots off potential places where it could be parametrised.